odxtools/comparamspec.py

```diff
@@ -51,8 +51,7 @@ def from_et(et_element: ElementTree.Element, doc_frags: List[OdxDocFragment]) ->
 
     def _build_odxlinks(self) -> Dict[OdxLinkId, Any]:
         odxlinks: Dict[OdxLinkId, Any] = {}
-        if self.odx_id is not None:
-            odxlinks[self.odx_id] = self
+        odxlinks[self.odx_id] = self
```

Comment:
Even in ODX 2.0 ?

Reply:
I just checked: in ODX 2.0, COMPARAM-SPEC is derived from ODX-CATEGORY for which the ID attribute is mandatory.

tl;dr: yes